### ISSUE TITLE
loss is much higher compared to provided log file

### ISSUE THREAD

**Issue by emon-altersense** [sming256/OpenTAD#74]
I am trying to train **adatad** on raw-video, thumos-14 dataset. I am facing the problem of much higher loss compared to given log https://drive.google.com/file/d/1esvBEVunTnK87Kh2g1dFZO-38zOGvn4-/view even though I have followed and installed everything including cuda, pytorch and other specified versions. My loss is saturated around 0.55 or sometimes even higher. I am using videomae-small as pretrained model.

CUDA: 11.8 (System wide install)
Pytorch: 2.0.1
mmcv: 2.0.1
mmaction2: 1.1.0

What is causing this type of behavior?
